notifications/README.md

```diff
@@ -0,0 +1,12 @@
+## Notifications
+
+- Notifications rely on [VAPID](https://web.dev/push-notifications-web-push-protocol/) keys
+- You can generate these keys using `openssl` as follows
+  ```
+  $ openssl ecparam -genkey -name prime256v1 -out private_key.pem
+  $ openssl ec -in private_key.pem -pubout -outform DER|tail -c 65|base64|tr -d '=' |tr '/+' '_-' >> public_key.txt
+  $ openssl ec -in private_key.pem -outform DER|tail -c +8|head -c 32|base64|tr -d '=' |tr '/+' '_-' >> private_key.txt
+  ```
+- Set `VAPID_PUBLIC_KEY` env. variable from `public_key.txt`
+- Set `VAPID_PRIVATE_KEY` env. variable from `private_key.txt`
+- Set `VAPID_ADMIN_EMAIL` env. variables accordingly
```

Comment:
What is the purpose of this setting? Who's email should be here? A Wagtail admin or a system administrator?

Reply:
System administrator, if push service provider find any problem they can inform us

[For reference: See here](https://blog.mozilla.org/services/2016/08/23/sending-vapid-identified-webpush-notifications-via-mozillas-push-service/#:~:text=subscription%20updates.-,Identifying%20Yourself,-Mozilla%20goes%20to)

Reply:
The pertinent part from that blog post is:

>...a mailto link for the administrative contact for this feed. It’s best if this email is not a personal email address, but rather a group email so that if a person leaves an organization, is unavailable for an extended period, or otherwise can’t respond, someone else on the list can. Mozilla will only use this if we notice a problem with your feed and need to contact you.

I assume that this does not only apply to Mozilla's push service but generally, no matter who provides service.

Would you update the readme to make it clear what the admin email is for?

Reply:
done